sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CProperties.java

```diff
@@ -116,13 +115,14 @@
 
     @Override
     public void afterPropertiesSet() {
-        if (StringUtils.isEmpty(tenant) && StringUtils.isEmpty(baseUri)) {
-            throw new AADB2CConfigurationException("'tenant' and 'baseUri' at least configure one item.");
-        }
-
-        if (!userFlows.keySet().contains(loginFlow)) {
-            throw new AADB2CConfigurationException("Sign in user flow key '"
-                + loginFlow + "' is not in 'user-flows' map.");
+        if (StringUtils.isEmpty(tenantId)) {
```

Comment:
Add proper comment here to indicate what does this condition `if (StringUtils.isEmpty(tenantId))` mean